tools/benchmark_test.py

```diff
@@ -9,9 +9,8 @@
 
 class TestBenchmark(DenoTestCase):
     def test_strace_parse(self):
-        with open(
-                os.path.join(sys.path[0], "testdata/strace_summary.out"),
-                "r") as f:
+        with open(os.path.join(sys.path[0], "testdata/strace_summary.out"),
+                  "r") as f:
```

Comment:
YAPF was upgraded from 0.23.0 to 0.30.0. I believe that's what's causing these changes.